CONTRIBUTING.md

```diff
@@ -59,9 +84,22 @@ workflow.
 5. Set line length to 120
 6. Install mypy plugin
```

Comment:
mypy, black, flake8, pylint - maybe drop a link to our .github/linters directory, so you don't have to maintain this part of the doc?

Reply:
update accepted and added with most recent commit